src/components/Receiving/const.js

```diff
@@ -2,4 +2,5 @@ export const PIECE_STATUS_EXPECTED = 'Expected';
 export const PIECE_STATUS_RECEIVED = 'Received';
 export const ITEM_STATUS = {
```

Comment:
Just curious... why define separate variables for the piece statuses, but an object for the item statuses?

Reply:
historically we had no info about available statuses, so one constant was fine. But importing one object with all statuses is more handy, so we might refactor piece statuses at some point in the future